sml_builder/method.py

```diff
@@ -13,6 +14,10 @@
     get_feature_config,
 )
 
+from .search_tool import search_partial
```

Comment:
Can you use an absolute import so that it is clearer where search_tool is imported from (i.e sml_builder.serch_tool). This also makes the code consistent with other imports in the file.

Reply:
No worries :)